src/components/DragCaptureZone/DragCaptureZone.jsx

```diff
@@ -65,24 +72,40 @@ const DragCaptureZone = createClass({
 	render() {
 		return (
 			<div
-				{...this.props}
+				{...omitProps(this.props, DragCaptureZone)}
 				className={cx('&', this.props.className)}
 				key='DragCaptureZone'
 				onMouseDown={this.handleDragStart}
+				ref={(ref) => {this.elementRef = ref;}}
 			/>
 		);
 	},
 
+	componentDidMount() {
+		//add event listeners directly on the DOM elemnt to allow preventDefault
+		//calls which are not honored due to react's event delegation
+		this.elementRef.addEventListener('touchstart', this.handleDragStart);
+		this.elementRef.addEventListener('touchmove', this.handleDrag);
+		this.elementRef.addEventListener('touchend', this.handleDragEnd);
+		this.elementRef.addEventListener('touchcancel', this.handleDragCancel);
+	},
+
 	componentWillUnmount() {
 		window.document.removeEventListener('mousemove', this.handleDrag);
 		window.document.removeEventListener('mouseup', this.handleDragEnd);
 	},
 
 	handleDrag(event) {
-		const {
-			pageX,
-			pageY,
-		} = event;
+		let pageX;
```

Comment:
What do we do with these variables? They just get assigned in this function but not used.